aws-rds-customdbengineversion/README.md

```diff
@@ -0,0 +1,12 @@
+# AWS::RDS::CustomDBEngineVersion
+
+Congratulations on starting development! Next steps:
+
+1. Write the JSON schema describing your resource, `aws-rds-customdbengineversion.json`
+1. Implement your resource handlers.
+
+The RPDK will automatically generate the correct resource model from the schema whenever the project is built via Maven. You can also do this manually with the following command: `cfn generate`.
+
+> Please don't modify files under `target/generated-sources/rpdk`, as they will be automatically overwritten.
+
+The code uses [Lombok](https://projectlombok.org/), and [you may have to install IDE integrations](https://projectlombok.org/setup/overview) to enable auto-complete for Lombok-annotated classes.
```

Comment:
Could you please re-generate this file?

Reply:
I tried many times. Seems that `cfn generate` not working. Many resources in our repo does not have README.md file generated. Can you try `cfn generate` on your end to confirm it is bug or only on my environment issue.